epictrack-web/package.json

```diff
@@ -79,6 +79,7 @@
     "eslint": "^8.39.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "prettier": "^2.8.8"
+    "prettier": "^2.8.8",
+    "prettier-eslint": "^16.3.0"
```

Comment:
I just added this dev dependency because the prettier eslint plugin in vs code stopped working and then I noticed it has this as a dependency